lib/tasks/superbolt.rake

```diff
@@ -1,9 +1,21 @@
 # assumes an environment task that sets up your app environment
 desc "worker that reads the queue and sends messages through the router as configured"
 task :superbolt => :environment do
+  statsd = if defined?(::Statsd) && Superbolt.statsd_host && Superbolt.statsd_port
+    ::Statsd.new(Superbolt.statsd_host, Superbolt.statsd_port)
+  else
+    nil
+  end
+
   begin
     Superbolt::App.new(Superbolt.app_name, {}).run do |message, logger|
-      Superbolt::Router.new(message, logger).perform
+      begin
+        Superbolt::Router.new(message, logger).perform
+        statsd.increment("app.#{Superbolt.app_name}.messages.#{message["event"]}.success") if statsd
```

Comment:
Should we name `superbolt` here?

Like for example going:

```
"superbolt.#{Superbolt.app_name}.messages.#{message['event']}.success"
```

?

Thinking what would be clearer to the user going through the metrics in the statsd UI...